plugins/modules/zypper.py

```diff
@@ -158,6 +158,13 @@
     description:
       - Adds C(--quiet) option to I(zypper) install/update command.
     version_added: '10.2.0'
+  skip_post_errors:
+    type: bool
+    required: false
+    default: false
+    description:
+      - When set to V(true), ignore zypper return code 107 (post install script errors).
+    version_added: '10.2.0'
```

Comment:
The next feature release will be 10.6.0:
```suggestion
    version_added: '10.6.0'
```

Reply:
Done